PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,24 +2,32 @@
 
 ## Description
 
-Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+Today we implemented a helper function called get_table, which selects tables based on their name. 
+This function is called upon in many ways. Instances include:
+- View data which updates and goes the first 5000 observations
+- deduplicate, which omits duplicates in a table
+- Future uses include add_to_use_count(), which will be used to track scripts and their effectiveness. 
+
+This code has eliminated 5 functions in the db.py file while maintaining the integrity of the application. 
+
+This pull request reduces the potential for bugs and adds scalability to the project. 
+explanation [video here](https://www.loom.com/share/26629a54700f4c1d89c4ce166377b58d) 
```

Comment:
I'm a little confused regarding the `PULL_REQUEST_TEMPLATE.md` file being added already filled out, rather than this additional context being provided in this actual PR's description.

A `PULL_REQUEST_TEMPLATE.md` is there to autofill the description box for any PR opened after its addition and having it filled out will cause confusion for future contributors.

I propose we revert this file to its default state, and add this info to this PRs description.